### ISSUE TITLE
Invoke method doesn't work

### ISSUE THREAD

**Issue by orlangur87** [adamhartford/SwiftR#24]
Hi! I'm getting this error while debugging using Safari: "SyntaxError: Unexpected token ')'"
```html
<!doctype html><html><head></head><body><script src='file:///Users/orlangur/Library/Developer/CoreSimulator/Devices/70F03748-4D1D-4F68-9FCB-6AFD1DBFA0DB/data/Containers/Data/Application/392AA7DA-1AC9-454E-951F-2295A433B4AF/tmp/jquery-2.1.3.min.js'></script><script src='file:///Users/orlangur/Library/Developer/CoreSimulator/Devices/70F03748-4D1D-4F68-9FCB-6AFD1DBFA0DB/data/Containers/Data/Application/392AA7DA-1AC9-454E-951F-2295A433B4AF/tmp/jquery.signalR-2.2.0.min'></script><script src='file:///Users/orlangur/Library/Developer/CoreSimulator/Devices/70F03748-4D1D-4F68-9FCB-6AFD1DBFA0DB/data/Containers/Data/Application/392AA7DA-1AC9-454E-951F-2295A433B4AF/tmp/SwiftR.js'></script>)</body></html>
```
That is because of this lines from lib:
```swift
let html = "<!doctype html><html><head></head><body>"
                + "\(jqueryInclude)\(signalRInclude)\(jsInclude))"
                + "</body></html>"
```
But my main issue that invoke method doesn't send messages. According server logs, I'm able to connect and disconnect, but not sending any requests. Here is my code:
```swift
SwiftR.connect("http://myDomain.com") { connection in
            connection.queryString = ["access_token": token]
            connection.connected = {
                self.hub?.invoke("getChats", arguments: nil, callback: { (result, error) -> () in
                       print("socket get chats: \(result) \(error)")
                })
            }
            self.hub = connection.createHubProxy("agenthub")
}
```
Thank you!

**Comment by adamhartford** [comment]
Looks like there's an extra `)` in there:

```swift
let html = "<!doctype html><html><head></head><body>"
                + "\(jqueryInclude)\(signalRInclude)\(jsInclude))"
                + "</body></html>"
```

should be:


```swift
let html = "<!doctype html><html><head></head><body>"
                + "\(jqueryInclude)\(signalRInclude)\(jsInclude)"
                + "</body></html>"
```

Are you using a version from CocoaPods or the repository head? There was recently a fix for calling server methods with no arguments that hasn't made it to CocoaPods yet.

**Comment by orlangur87** [comment]
I tried with CocoaPods and now either using sources from repository. Still have same result: 
connection created -> hub created -> connection started -> "connected" block executed. 
Then I tried to invoke (tried with and without parameters), but completion callback not called.
```Swift
func tryToInvoke() {
        print("trying") // that appears in console
        hub.invoke("method", arguments: ["arg"]) { (result, error) -> () in
            print("callback") // and that is not
        }
    }
```

**Comment by adamhartford** [comment]
Can you check the Safari console again? Set up some action in your app so that you can try invoking a SignalR server method at any time, then see if any errors are reported.

**Comment by orlangur87** [comment]
Now I'm getting this error: 
TypeError: undefined is not an object (evaluating 'swiftR.hubs.agentHub.invoke') 
(anonymous function)test.app:1
Although I'm sending invoke with argument string.

**Comment by adamhartford** [comment]
I see the problem. Some things behind the scenes haven't been hooked up because you haven't defined any handlers. I'll make some changes such that it'll work the way you're trying now, but for now try adding some handler (even if it's just a dummy handler):

```swift
self?.hub = connection.createHubProxy("agenthub")
self?.hub.on("someEvent") { _ in } // <-- Add this line
```

I haven't yet considered a use case where you don't have a handler for a server event, so the code expects there to be at least one.

**Comment by adamhartford** [comment]
Now you shouldn't need to add the line from my comment above. Everything should work as is.

**Comment by adamhartford** [comment]
Fixed in v0.9.0.
